openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Customproperties.spec.ts

```diff
@@ -297,16 +298,425 @@ describe('Custom Properties should work properly', { tags: 'Settings' }, () => {
+          customType: 'Entity Reference List',
+          entityReferenceConfig: entity.entityReferenceConfig,
+        });
+
+        // Navigating back to custom properties page
+        cy.settingClick(entity.entityApiType, true);
+
+        verifyResponseStatusCode('@getEntity', 200);
+      });
+
+      it(`Edit created property for ${entity.name} entity`, () => {
+        interceptURL(
+          'GET',
+          `/api/v1/metadata/types/name/${entity.name}*`,
+          'getEntity'
+        );
+
+        // Selecting the entity
+        cy.settingClick(entity.entityApiType, true);
+
+        verifyResponseStatusCode('@getEntity', 200);
+        editCreatedProperty(propertyName, 'Entity Reference List');
+      });
+
+      it(`Delete created property for ${entity.name} entity`, () => {
+        interceptURL(
+          'GET',
+          `/api/v1/metadata/types/name/${entity.name}*`,
+          'getEntity'
+        );
+
+        // Selecting the entity
+        cy.settingClick(entity.entityApiType, true);
+
+        verifyResponseStatusCode('@getEntity', 200);
+        deleteCreatedProperty(propertyName);
+      });
+    });
+  });
+
+  describe('Add update and delete Date custom properties', () => {
+    Object.values(ENTITIES).forEach((entity) => {
+      const propertyName = `addcyentity${entity.name}test${uuid()}`;
+
+      it(`Add Date custom property for ${entity.name} Entities`, () => {
+        interceptURL(
+          'GET',
+          `/api/v1/metadata/types/name/${entity.name}*`,
+          'getEntity'
+        );
+
+        // Selecting the entity
+        cy.settingClick(entity.entityApiType, true);
+
+        verifyResponseStatusCode('@getEntity', 200);
+
+        addCustomPropertiesForEntity({
+          propertyName,
+          customPropertyData: entity,
+          customType: 'Date',
+          formatConfig: entity.dateFormatConfig,
+        });
+
+        // Navigating back to custom properties page
+        cy.settingClick(entity.entityApiType, true);
+
+        verifyResponseStatusCode('@getEntity', 200);
+      });
+
+      it(`Edit created property for ${entity.name} entity`, () => {
+        interceptURL(
+          'GET',
+          `/api/v1/metadata/types/name/${entity.name}*`,
+          'getEntity'
+        );
+
+        // Selecting the entity
+        cy.settingClick(entity.entityApiType, true);
+
+        verifyResponseStatusCode('@getEntity', 200);
+        editCreatedProperty(propertyName);
+      });
+
+      it(`Delete created property for ${entity.name} entity`, () => {
+        interceptURL(
+          'GET',
+          `/api/v1/metadata/types/name/${entity.name}*`,
+          'getEntity'
+        );
+
+        // Selecting the entity
+        cy.settingClick(entity.entityApiType, true);
+
+        verifyResponseStatusCode('@getEntity', 200);
+        deleteCreatedProperty(propertyName);
+      });
+    });
+  });
+
+  describe('Add update and delete DateTime custom properties', () => {
```

Comment:
Not sure if it's possible to create an array of types and just loop over it instead writing each describe block separately